package.json

```diff
@@ -2,19 +2,21 @@
   "name": "ping-centre",
   "version": "1.0.0",
   "description": "A client for easily collecting events and metrics.",
-  "main": "ping-centre.js",
+  "main": "src/ping-centre.js",
```

Comment:
I suppose it depends how the user wants to use this. ping-centre should work in node and also in the browser so both make sense I think